quiche/include/quiche.h

```diff
@@ -369,6 +369,9 @@ void quiche_conn_destination_id(quiche_conn *conn, const uint8_t **out, size_t *
 void quiche_conn_application_proto(quiche_conn *conn, const uint8_t **out,
                                    size_t *out_len);
 
+/// Returns the peer's leaf certificate (if any) as a DER-encoded buffer.
+void quiche_conn_session(quiche_conn *conn, const uint8_t **out, size_t *out_len);
```

Comment:
Should've been `quiche_conn_peer_cert` right? copy and paste mistake probably!

Reply:
Whoops